src/presign/mod.rs

```diff
@@ -19,8 +19,8 @@
 mod input;
 mod participant;
 mod record;
-mod round_one;
-mod round_three;
+pub(crate) mod round_one;
```

Comment:
These two modules were made pub(crate) since the Secret* types are defined in them. As we talked about, I think this is the wrong place for these types and we should move them. Once they are moved, this change can be reverted (will make a separate comment for that).